docs/index.md

```diff
@@ -2,46 +2,50 @@
 
 # DSP-TOOLS documentation
 
-DSP-TOOLS is a Python package with a command line interface that helps you interact with a DSP server. The DSP server 
```

Comment:
The changes in this file attempt to make the landing page on https://pypi.org/project/dsp-tools/ more readable. From line 22 on, it was for example necessary to put the full links to the docs pages, instead of relative links.